src/core/js/models/adaptModel.js

```diff
@@ -236,9 +245,9 @@ define([
         },
 
         getAvailableChildren: function() {
-            return new Backbone.Collection(this.getChildren().where({
+            return this.getChildren().where({
```

Comment:
we're going to need two versions of ```getAvailableChildren``` and ```findDescendants```. the original with a deprecated message and a new version we can switch over to. otherwise this is gonna cause some wonderful breakages in 3rd party extensions.

this is a rather large breaking change otherwise.